README.md

```diff
@@ -1 +1,127 @@
-# vault-plugin-auth-jwt
+# Vault Plugin: JWT Auth Backend [![Build Status](https://travis-ci.org/hashicorp/vault-plugin-auth-jwt.svg?branch=master)](https://travis-ci.org/hashicorp/vault-plugin-auth-jwt)
+
+This is a standalone backend plugin for use with [Hashicorp Vault](https://www.github.com/hashicorp/vault).
+This plugin allows for JWTs (including OIDC tokens) to authenticate with Vault.
```

Comment:
// , Does this imply that we could log in with Generic SAML or Ping?

Reply:
If your SAML provider will give you a JWT, yes.